dist/amd/dialog-renderer.js

```diff
@@ -126,14 +159,7 @@ define(['exports', 'aurelia-templating'], function (exports, _aureliaTemplating)
       controller.centerDialog = function () {
         var child = modalContainer.children[0];
 
-        var vw = Math.max(document.documentElement.clientWidth, window.innerWidth || 0);
-        child.style.marginLeft = Math.max((vw - child.offsetWidth) / 2, 0) + 'px';
-
-        if (!settings.centerHorizontalOnly) {
-          var vh = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
-
-          child.style.marginTop = Math.max((vh - child.offsetHeight) / 2, 30) + 'px';
-        }
+        if (!settings.centerHorizontalOnly) {}
```

Comment:
No. I am using flex. I have a way to do it in CSS without flex for IE9 as
well, I'll make the PR with the example tomorrow.
On Wed, Oct 28, 2015 at 6:47 AM Patrick Walters <notifications@github.com>
wrote:

> In dist/amd/dialog-renderer.js
> <https://github.com/aurelia/dialog/pull/50#discussion_r43256561>:
>
> > @@ -126,14 +159,7 @@ define(['exports', 'aurelia-templating'], function (exports, _aureliaTemplating)
> >        controller.centerDialog = function () {
> >          var child = modalContainer.children[0];
> >
> > -        var vw = Math.max(document.documentElement.clientWidth, window.innerWidth || 0);
> > -        child.style.marginLeft = Math.max((vw - child.offsetWidth) / 2, 0) + 'px';
> > -
> > -        if (!settings.centerHorizontalOnly) {
> > -          var vh = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
> > -
> > -          child.style.marginTop = Math.max((vh - child.offsetHeight) / 2, 30) + 'px';
> > -        }
> > +        if (!settings.centerHorizontalOnly) {}
>
> Is this not needed anymore to center?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aurelia/dialog/pull/50/files#r43256561>.
>